anime-list-master.xml

```diff
@@ -49168,7 +49168,7 @@
   <anime anidbid="17810" tvdbid="movie" defaulttvdbseason="1" episodeoffset="" tmdbid="1142115" imdbid="">
     <name>Eiga Sore Ike! Anpanman: Roboly to Pokapoka Present</name>
   </anime>
-  <anime anidbid="17811" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="17811" tvdbid="movie" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt28055106">
```

Comment:
```suggestion
  <anime anidbid="17811" tvdbid="movie" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="tt28055106">
```

Reply:
Fine with me, The reasoning for the defaulttvdbseason is that if this is added as a special to tvdb it will be season 0; and as far as I remember there are already both 1 and 0 for defaulttvdbseason for non-tvdb-entries, but it doesn't really matter for me :slightly_smiling_face:

Reply:
To this day I still don't understand why the tvdb season is set at all in these situations. Just trying to keep it consistent with the `1` by default